### PR TITLE
PK Mod: Doom items

### PR DESCRIPTION
**Pull request by pisskop**
Adds item lists for Doom Critters.

-Pokeballs, for funzies
-Doom Armors, Doom rockets
-Melee weapons item drops for humies

Dem pokeballs, with a little code they could actually function like the real thing.

### CODE REVIEW

File: data/mods/PKs_Rebalance/pk_critter_robot.json [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    "dodge":2,
+    "armor_bash":4,
+    "armor_cut":10,
+    "vision_day":40,
+    "special_attacks":[["KAMIKAZE", 0]],
+    "starting_ammo": { "grenade_inc": 1},
+    "luminance":3,
+    "hp":15,
+    "death_function":["EXPLODE"],
+    "description":"This manhack, almost twice the size of normal manhacks, carries an incindiary explosive within its core.  It is designed to fight until its destruction and then explode rather than explode in proximitry.",
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_critter_robot.json]: incindiary -> incendiary

**Comment by pisskop** [comment]
Aye.  Thank ya.

But they dont work as is because they rely on explosives that utilize iuse.

**Comment by pisskop** [comment]
Im having trouble getting th 'beserker' effect,

"id": "doom_stimpack",

  
To 'step down', instead it seems to fade away entirely.  Is there a way to make it fade over time, say 10, 15 minutes?

File: data/mods/PKs_Rebalance/pk_item_armor.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        "type" : "ARMOR",
+        "id" : "megaarmor_torso_1",
+        "category" : "armor",
+        "name" : "space marine armor",
+        "weight" : 3580,
+        "color" : "ltgreen",
+        "covers" : ["TORSO"],
+        "to_hit" : -2,
+        "symbol" : "[",
+        "description" : "A green piece of PRM armor, made from microlayered polymer plastic and tapering off into a thin, flexible filiment at the shoulder, collarbone, and stomach.  Offers decent protection from flame and acid, and features various straps and pockets for holstering guns and holding stims and ammo.",
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_item_armor.json]: filiment -> filament

File: data/mods/PKs_Rebalance/pk_item_armor.json [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        "id" : "megaarmor_boots_1",
+        "category" : "armor",
+        "name" : "space marine boots",
+        "name_plural" : "space marine boots",
+        "weight" : 1190,
+        "color" : "ltgray",
+        "covers" : ["FEET"],
+        "to_hit" : -1,
+        "symbol" : "[",
+        "description" : "Lovingly refered to as shitstompers, these PRM polyplastic boots reach up to the knee, where they are sealed with a large strap.  They can be very hard to even walk around in untrained, especially if they are imporperly fitted.",
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_item_armor.json]: imporperly -> improperly

File: data/mods/PKs_Rebalance/pk_item_tools.json [hunk 4]
```diff
@@ -0,0 +1,290 @@
+[
+
+    {
+    "id": "pokeball",
+    "type": "TOOL",
+    "name": "pocket ball",
+    "color": "light_red",
+    "symbol": "o",
+    "description": "A fist-sized ball that holds the . . . erm, 'energy' of a monster, allowing you to transport them within safely.",
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_item_tools.json]: . . . -> ...

File: data/mods/PKs_Rebalance/pk_item_tools.json [hunk 5]
```diff
@@ -0,0 +1,290 @@
...
+    "id" : "medikit",
+    "name" : "medikit",
+    "weight" : 360,
+    "color" : "red",
+    "comestible_type" : "MED",
+    "symbol" : "+",
+    "quench" : 0,
+    "healthy" : -5,
+    "nutrition" : 0,
+    "description" : "A PRM brand medikit.  Can be quickly applied in battlefield situations, though its designed to quickly stopgap wounds with artifical coagulants, not disinfect.",
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_item_tools.json]: artifical -> artificial

File: data/mods/PKs_Rebalance/pk_item_tools.json [hunk 6]
```diff
@@ -0,0 +1,290 @@
...
+            {
+                "id": "pkill1",
+                "duration": 150
+            } 
+        ]
+    }  
+    },{
+    "type" : "TOOL_ARMOR",
+    "id" : "beserker_drug",
+    "name" : "beserker stimpack",
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_item_tools.json]: beserker -> berserker

File: data/mods/PKs_Rebalance/pk_item_tools.json [hunk 7]
```diff
@@ -213,5 +213,37 @@
             "duration": 40
         }    
     ]}
+    },{
+    "type" : "COMESTIBLE",
+    "id" : "stamina_vial",
+    "name" : "stamina vial",
+    "volume" : 3,
+    "description" : "Containing a slew of charged organic chemicals, this 3-layer slurry might go down hard but provides a noticable endurance boost for hardcore athletes and demanding soldiers alike.  Like most PRM products it is designed for short-term use only, and may upset the body's balance.",
```
Comment (cainiaowu) [inline data/mods/PKs_Rebalance/pk_item_tools.json]: noticable -> noticeable

**Comment by Rivet-the-Zombie** [comment]
>-Pokeballs, for funzies

I thought this was Doom, not Pokemon.

**Comment by pisskop** [comment]
:<

**Comment by Coolthulhu** [comment]
Errors on load

**Comment by pisskop** [comment]
righto

**Pull request closed without merging**